### DISCUSSION TITLE
Configurability of Evidence file locations

### DISCUSSION THREAD

**Discussion by f8dca** [evidence-dev/evidence#2388] [General]
## Request Type

- [ ] Small Fix (wording, errors, typos, etc.)
- [x] Explanation Improvement (make a description clearer for the reader of the docs)
- [ ] Walkthrough Request (new tutorial for a particular feature or use case)
- [ ] Other

## Description

What is the recommended folder structure for multiple projects. ideally I do not want folders with individual node installations for each of my projects.

**Comment by archiewood** [comment]
Multiple evidence projects inside one repo?

Or evidence and other data tools in one repo?

**Comment by f8dca** [comment]
I would like to separate everything  (tool, input, output, config)

Perhaps something like this ?  

`npm run build --source c:\src --dest e:\dest --conf f:\cnf.yaml`


- **source**  = directory for the markdown files
- **dest**  = directory for the output from the build command
- **conf**  = configuration for the site DB and others

**Comment by archiewood** [comment]
aha, that's definitely a feature request - this is not _easily_ doable out of the box!
